src/Operation/ListIndexes.php

```diff
@@ -73,7 +73,7 @@ class ListIndexes implements Executable
      * @param array  $options        Command options
      * @throws InvalidArgumentException for parameter/option parsing errors
      */
-    public function __construct($databaseName, $collectionName, array $options = [])
+    public function __construct(string $databaseName, string $collectionName, array $options = [])
```

Comment:
It just occurred to me that after adding `string` type hints to `$databaseName` and `$collectionName` (among other parameters), there's an opportunity to clean up the explicit casting that often follows. For instance, the end of this constructor has:

```php
$this->databaseName = (string) $databaseName;
$this->collectionName = (string) $collectionName;
```

Maybe do a search for `(string)` or `(int)` casts throughout the library and see if any of those are now redundant.

Reply:
Good point. I removed these using PhpStorm, which offers an inspection for unnecessary type casts.